app/mailers/grouped_issue_mailer.rb

```diff
@@ -1,39 +1,50 @@
 class GroupedIssueMailer < ApplicationMailer
+  include ActionView::Helpers::TextHelper
   default from: "Epic Logger <admin@epiclogger.com>"
   def notify_subscriber(group, user, sender, message)
     @sender = sender
     @group = group
     @message = message
-    @website = @group.website
     @user = user
     mail to: @user.email, subject: 'Epic Logger Subscriber notification'
   end
 
-  def error_occurred(issue)
+  def error_occurred(issue, member)
     @issue = issue
-    @website = issue.website
-    mail to: website_owners_emails(@website), subject: 'Epic Logger Realtime Error'
+    @member = member
+    mail to: @member.user.email, subject: "Project: [#{@issue.website.title}] #{truncate(@issue.message, length: 40)}"
```

Comment:
.... @Panioglo 💩  ... :(

The subject should be: `[#{@issue.website.title}] #{truncate(@issue.message, length: 40)}`